ext/standard/uniqid.c

```diff
@@ -72,12 +74,17 @@ PHP_FUNCTION(uniqid)
 	gettimeofday((struct timeval *) &tv, (struct timezone *) NULL);
 	sec = (int) tv.tv_sec;
 	usec = (int) (tv.tv_usec % 0x100000);
+	php_random_bytes(bytes, UNIQID_RAND_CHARS*2, 1);
+	for (i = 0; i < UNIQID_RAND_CHARS; i++) {
```

Comment:
@CarwynNelson 
uniqid() produces timestamp based unique ID. This is what it supposed to do. (AFAIK, it was made for SMTP's message ID which uses timestamp based ID) 
"more entropy" option should do better job. This RFC improves "more entropy" option by replacing php_combined_lcg() by php_random_bytes().